rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala

```diff
@@ -1053,10 +1053,8 @@ class ReduceSpec extends FlatSpec with Matchers with PersistentStoreTester {
 
   "variable references" should "be substituted before being used." in {
     implicit val errorLog = new ErrorLog()
-    val costAccounting =
-      CostAccountingAlg.monadState(AtomicRefMonadState.of[Task, CostAccount](CostAccount.zero))
-    val splitRandResult = rand.splitByte(3)
-    val splitRandSrc    = rand.splitByte(3)
+    val splitRandResult   = rand.splitByte(3)
+    val splitRandSrc      = rand.splitByte(3)
```

Comment:
thanks for ggetting rid of that `val costAccounting` boilerplate in each test :)

Reply:
Well, it's still there I think  :)

Reply:
I will replace with the reference to empty cost accounting.

Reply:
Yeah :D

why empty? Isn't zero the default state of a cost account?

Reply:
Also, while we're at it - wouldn't defining a class-level `val costAccounting = costAccountingZero` and deleteing the vals inisde tests get rid of (some of) the boilerplate?

Reply:
No, because we don't want to share the `CostAccounting` state between  test cases.

Reply:
I missed the AtomicRef in the test, so yeah...